### PR TITLE
Add CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by philipbrito**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Contributing Guidelines - Rocket.Chat.Java.SDK
+
+Great to have you here! Here are a few ways you can help make this project better!
+
+## Reporting an Issue
+
+[Github Issues](https://github.com/RocketChat/Rocket.Chat.Android/issues) are used to track todos, bugs, feature requests, and more.
```
Comment (luciofm) [inline CONTRIBUTING.md]: You are pointing to the Android repository

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+1. A Java Integrated Development Environment (e.g. IntelliJ IDEA, Eclipse IDE for Java Developers or NetBeans IDE).
+2. Clone this repo:
+```
+git clone https://github.com/RocketChat/Rocket.Chat.Java.SDK
+```
+3. Build the project.
+
+### Code style guide
+
+Before submitting a PR you should follow our [Coding Style](https://github.com/RocketChat/Rocket.Chat.Java.SDK/blob/develop/CODING_STYLE.md).
```
Comment (luciofm) [inline CONTRIBUTING.md]: You need to add the coding style file, or point to the one on the Android repository

**Review by luciofm — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 2 above)
Comment (philipbrito) [reply CONTRIBUTING.md]: Blocked from #40

File: CONTRIBUTING.md (hunk 1 above)
Comment (philipbrito) [reply CONTRIBUTING.md]: Fixed

File: CONTRIBUTING.md (hunk 2 above)
Comment (luciofm) [reply CONTRIBUTING.md]: @filipedelimabrito can you just point to https://github.com/RocketChat/java-code-styles/blob/master/CODING_STYLE.md

**Review by luciofm — APPROVED** [review]

**Pull request merged**